tests/lib/tools/tests.session

```diff
@@ -266,6 +266,7 @@ main() {
 		--pipe \
 		--service-type simple \
 		--setenv TERM=xterm-256color \
+		--setenv XDG_RUNTIME_DIR=$XDG_RUNTIME_DIR \
```

Comment:
Is this relevant to the PR?

Reply:
no, I'll remove it. Thanks for the review